scripts/knot_benchmark/src/benchmark/cases.py

```diff
@@ -134,6 +134,7 @@ def cold_command(self, project: Project, venv: Venv) -> Command:
         command = [
             str(self.path),
             "--venvpath",
+            "--threads",
```

Comment:
If I run `uv run benchmark` with `-vv` locally, it's clear that pyright is no longer resolving any third-party types installed into the virtual environment, because `--venvpath` requires a value:

```suggestion
            "--threads",
            "--venvpath",
```

Reply:
```
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/cache.py:12:6 - error: Import "platformdirs" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/cache.py:14:6 - error: Import "_black_version" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/cache.py:144:59 - error: Expression of type "dict[str, tuple[float | int | str, ...]]" is incompatible with declared type "Dict[str, Tuple[float, int, str]]" (reportAssignmentType)
/private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/concurrency.py
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/concurrency.py:18:6 - warning: Import "mypy_extensions" could not be resolved from source (reportMissingModuleSource)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/concurrency.py:34:16 - error: Import "uvloop" could not be resolved (reportMissingImports)
/private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:20:6 - warning: Import "mypy_extensions" could not be resolved from source (reportMissingModuleSource)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:21:6 - error: Import "packaging.specifiers" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:22:6 - error: Import "packaging.version" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:23:6 - error: Import "pathspec" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:24:6 - error: Import "pathspec.patterns.gitwildmatch" could not be resolved (reportMissingImports)
  /private/var/folders/gd/1czdxc454j15y8gns2krv8vc0000gn/T/tmpm3ut7zl3/src/black/files.py:42:12 - warning: Import "colorama" could not be resolved from source (reportMissingModuleSource)
```

Reply:
I filed #13228